public/javascripts/controllers/farmer-ctrl.js

```diff
@@ -38,10 +38,10 @@ angular.module('jasmic.controllers')
      */
     .controller('FarmerProfileCtrl', ['$scope', '$location', '$routeParams', '$mdDialog', 'OpenTransactionsFactory',
         'TransactionsFactory', 'FarmerFactory', 'ParishesFactory', 'FarmerFarmFactory', 'CropsFactory',
-        'UnitsFactory', 'CommodityFactory', 'CommoditiesFactory', 'DistrictsFactory', 'FarmerMembershipsFactory',
+        'UnitsFactory', 'CommodityFactory','CommoditiesFactory', 'DistrictsFactory', 'FarmerMembershipsFactory','CommodityEditFactory', 'FarmEditFactory',
```

Comment:
@nick22891 Reviewing the develop branch, I'm seeing where the FarmerFarmFactory handles the updating of a Farm. You can use that instead of creating another Factory. An aside, before trying to tackle the solution to a problem see if there is any related code in the code base that handles that specific functionality. Reuse, Reuse, Reuse!